ipi/engine/motion/dynamics.py

```diff
@@ -317,16 +317,16 @@ def pconstraints(self):
             p = dstrip(self.beads.p)
             m = dstrip(self.beads.m3)[:, 0:na3:3]
             M = self.beads[0].M
+            Mnb = M*nb
 
+            dens = 0
             for i in range(3):
-                pcom[i] = p[:, i:na3:3].sum()
+                pcom = p[:, i:na3:3].sum()
```

Comment:
Looks like with these changes, line 313 above:
```
pcom = np.zeros(3, float)
```
is not needed.

Reply:
agreed.